frontend/src/my-books/myBookModal/ShelfModal.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+The book project lets a user keep track of different books they would like to read, are currently
+reading, have read or did not finish.
+Copyright (C) 2020  Karan Kumar
+
+This program is free software: you can redistribute it and/or modify it under the terms of the
+GNU General Public License as published by the Free Software Foundation, either version 3 of the
+License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with this program.
+If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import React, { Component } from 'react'
+import Modal, {IModalProps} from './Modal'
```

Comment:
Import in our Modal here instead (found in the `components` folder)

Reply:
The modal in the components folder has a fixed width of 380px which looks too big for a smaller screen. I had to change the width to 50%. This has affected the width of the change password modal. Please let me know if there are better approaches to take.

Reply:
The 380 set is not for mobile design but rather for the base Modal itself on desktop. To get these modals to be responsive, we need to use Material-UI's `useMediaQuery`hook. I have successfully got this working on my end so all modals are now responsive. 
  For now, you can leave your code as is and I will go back into my branch (after we get your branch reviewed and merged) to make necessary changes to the base Modal.

Reply:
I have started your review and will get back in touch with you soon :)

Reply:
@olusolaa Hello! I added the fix for the modal in my PR for another ticket. We will work on getting that merged first so you can then take in those changes. Hopefully once that happens, the modal will be responsive on your end and you can double-check that the rest of your responsive styles still work.